src/herbivoresAndCarnivores.test.js

```diff
@@ -100,7 +100,7 @@
     expect(Animal.alive.includes(bear))
       .toBe(true);
 
-    expect(Animal.alive.includes(pig))
+git    expect(Animal.alive.includes(pig))
```

Comment:
There is a typo here: 'git' should not be present. Remove it to ensure the test runs correctly.